libs/queue/src/main.sw

```diff
@@ -0,0 +1,98 @@
+// The `Queue` type corresponds to the same data structure.
+library;
+
+use std::vec::Vec;
+
```

Comment:
```suggestion
```

The vector comes with the prelude therefore we do not need to import it ourselves.

Reply:
Removed